src/Renderer/SpriteRenderer.js

```diff
@@ -60,26 +60,36 @@ function(      WebGL,         glMatrix,      Camera )
 			mat[3].y += mat[0].y * x + mat[1].y * y + mat[2].y * z;
 			mat[3].z += (mat[0].z * x + mat[1].z * y + mat[2].z * z) + (uCameraLatitude * floor(min(uCameraZoom, 1.0)) / 50.0);
```

Comment:
Regarding my last comment, what happens if you comment this out?

Reply:
Changed to `			mat[3].z += (mat[0].z * x + mat[1].z * y + mat[2].z * z);`
If i comment it out entirely it just projects wrong the sprites completely

![f913d58584f08221e8e27b3657db34e4](https://github.com/MrAntares/roBrowserLegacy/assets/4583730/b7fa8ef5-cfe4-4d5a-bea4-5b90ce41d4a2)
![12004c054d0dc7fb2c164c8a29dbe419](https://github.com/MrAntares/roBrowserLegacy/assets/4583730/a626e088-5795-4c74-b822-0558a12b02ef)